tests.toml

```
@@ -0,0 +1,30 @@
+test_format = 1.0
+
+[default]
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    args.email = "john@gmail.com"
+    args.admin_password = "admin!42"
+    args.version = "Enterprise"
+	args.team_display_name = "Mon équipe"
+
+    # -------------------------------
+    # Commits to test upgrade from
+    # -------------------------------
+
+    test_upgrade_from.1677cb9f.name = "Upgrade from 7.3.0"
```

I think we'd want to test from 7.2.1 too ; this is the version that was the most widely deployed before the migration.

---

And actually version 7.3. wasn't widely deployed: we could even just replace this line by the version 7.2, which should catch the majority of migrations from older systems

```suggestion
    test_upgrade_from.4530e34.name = "Upgrade from 7.2.0"
```